src/matrix.js

```diff
@@ -1476,9 +1512,6 @@ export default class Matrix extends AbstractMatrix {
 
   removeRow(index) {
     checkRowIndex(this, index);
-    if (this.rows === 1) {
-      throw new RangeError('A matrix cannot have less than one row');
-    }
```

Comment:
I'd keep a check to avoid a negative number of rows

Reply:
that can't happen due to the `checkRowIndex` precondition (and same for `checkColumnIndex` in other locations).

I can add if you'd prefer defensive code.

Reply:
Oh right. it's fine if `checkRowIndex` already verifies it 👍